portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
Looks like this crossed wires with your other PR.

Reply:
Yeah I wasn't sure if I should created a separate branch since the week-3-tutorial was dependent on uploading the project to GCP.